client/src/locales/fr-FR.json

```diff
@@ -55,7 +55,7 @@
         },
         "none": "Non partagé",
         "members": "Membre ({count}) | Membres ({count})",
-        "suggestion": "Suggestions d'utilisateurs"
+        "suggestion": "Suggestions"
```

Comment:
English term "Suggested" is a verb, not a noun.

French translation should be something like "Suggérés", "Proposées" ou "Recommandés"